pkg/resources/marketplace/manifestHelper.go

```diff
@@ -6,18 +6,37 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"reflect"
 	"regexp"
 	"strconv"
 	"strings"
 
 	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
 )
 
 const (
 	manifestEnvVarKey  = "MANIFEST_DIR"
 	defaultManifestDir = "manifests"
 )
 
+type CrdConfig struct {
```

Comment:
Hmm, have you tried importing the CRD type from:
https://github.com/kubernetes/apiextensions-apiserver/blob/4f04d5ce70bb4470e352582ac2d039148b233cd1/pkg/apis/apiextensions/v1/types.go#L359 ?
I am fine with keeping this as is, just curious if there was something that prevented the import (version of the type maybe?).

Reply:
I had tried to use that but it wasn't working for me. I've just retried and it's working now so I'll switch and push a new update. I was getting an error when debugging which later turned out to be a vscode bug rather than code bug.